docs/source/en/model_doc/electra.md

```diff
@@ -14,66 +14,96 @@ rendered properly in your Markdown viewer.
+
+This approach is super efficient because ELECTRA learns from every single token in the input, not just the masked ones. That's why even the small ELECTRA models can match or outperform much larger models while using way less computing resources.
+
+You can find all the original ELECTRA checkpoints under the [ELECTRA release](https://huggingface.co/collections/google/electra-release-64ff6e8b18830fabea30a1ab) collection.
+
+> [!TIP]
+> Click on the right sidebar for more examples of how to use ELECTRA for different language tasks like sequence classification, token classification, and question answering.
+
+The example below demonstrates how to use ELECTRA for text classification tasks with [`Pipeline`] or the [`AutoModel`] class.
+
+<hfoptions id="usage">
+<hfoption id="Pipeline">
+
+```py
+from transformers import pipeline
+
+classifier = pipeline("text-classification", model="google/electra-small-discriminator")
+result = classifier("This restaurant has amazing food!")
+print(result)
+```
+
+</hfoption>
+<hfoption id="AutoModel">
+
+```py
+from transformers import AutoTokenizer, AutoModelForSequenceClassification
+import torch
+tokenizer = AutoTokenizer.from_pretrained("google/electra-small-discriminator")
+model = AutoModelForSequenceClassification.from_pretrained("google/electra-small-discriminator")
+inputs = tokenizer("ELECTRA is more efficient than BERT", return_tensors="pt")
+with torch.no_grad():
+outputs = model(**inputs)
+logits = outputs.logits
+print(logits)
+```
+
+</hfoption>
+</hfoptions>
+
+Quantization reduces the memory burden of large models by representing the weights in a lower precision. Refer to the [Quantization](../quantization/overview) overview for more available quantization backends.
+
+The example below uses [torchao](../quantization/torchao) to quantize the weights to int4:
+
+```py
+# pip install torchao
+import torch
+from transformers import TorchAoConfig, AutoModelForSequenceClassification, AutoTokenizer
+
+quantization_config = TorchAoConfig("int4_weight_only", group_size=128)
+model = AutoModelForSequenceClassification.from_pretrained(
+    "google/electra-large-discriminator",
+    torch_dtype=torch.bfloat16,
+    device_map="auto",
+    quantization_config=quantization_config
+)
+tokenizer = AutoTokenizer.from_pretrained("google/electra-large-discriminator")
+inputs = tokenizer("ELECTRA uses less compute than other models", return_tensors="pt").to("cuda")
+with torch.no_grad():
+    outputs = model(**inputs)
+    logits = outputs.logits
+print(logits)
+```
+
+## Notes
+
+- ELECTRA consists of two transformer models: a generator (G) and a discriminator (D). For most downstream tasks, use the discriminator model (`*-discriminator`) rather than the generator.
```

Comment:
```suggestion
- ELECTRA consists of two transformer models, a generator (G) and a discriminator (D). For most downstream tasks, use the discriminator model (as indicated by `*-discriminator` in the name) rather than the generator.
```